src/assets/download-icon.svg.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+  <svg class="sc-message--file-download-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50" width="30px" height="30px" @click="$emit('download')">
```

Comment:
I don't think the `@click` should be handled here, maybe better to let the user of this component to decide --> `<DownloadIcon @click="$emit('download')" />`. Do you agree?